### DISCUSSION TITLE
how to address a vulnerability to a maintainer

### DISCUSSION THREAD

**Discussion by remh8er** [community/community#129487] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all, 

I just started with github. 
Yesterday I cloned a  repository to use for my own project.
The code is not maintained for four years, but there are 28 forks and the repository has 43 stars. One of the forks is last update 4 months ago, So I thought this must be oke!

After I run the project locally I found a XSS vulnerabillity (reflected) and a SQL injection.
I have read all the docs on github how to report a vulnerability.

```
Note: If the repository doesn't have private vulnerability reporting enabled, you need to initiate
the reporting process by following the instructions in the security policy for the repository,
or create an issue asking the maintainers for a preferred security contact. For more information
,see "About coordinated disclosure of security vulnerabilities."

```

**Situation:**
The owner has no security policy enabled. 
On github I have no contact information and I used Linkedin to reached out.
After 3 weeks still no response. 

**Question**
What can I do more, without publically post my findings?

Kind Regards,

R.

**Comment by supermaxstar** [comment]
Hi,
Given the repository owner’s lack of response and the absence of a security policy, you can try reaching out to active contributors or maintainers of the more recently updated forks. 
They might be able to address the vulnerabilities or help you get in touch with the original owner. 
Additionally, consider opening a non-detailed issue in the original repository, politely requesting that the owner or maintainers set up a security policy or provide a contact for reporting vulnerabilities. 
Please try!
Hope this will help you.
Thank you for your attention!

**Comment by remh8er** [comment]
Thanks supermaxstar,

Will do!
